include/bluetooth/conn.h

```diff
@@ -209,6 +211,22 @@ int bt_conn_disconnect(struct bt_conn *conn, u8_t reason);
 struct bt_conn *bt_conn_create_le(const bt_addr_le_t *peer,
 				  const struct bt_le_conn_param *param);
 
+/** @brief Automatically connect to remote devices in whitelist.
+ *
+ *  This uses the Auto Connection Establishment procedure.
+ *
+ *  @param param Initial connection parameters.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ */
+int bt_conn_create_auto_le(const struct bt_le_conn_param *param);
```

Comment:
So will it work similarly to direct advertising?
i.e. we would set whitelist and call `bt_conn_create_auto_le` instead of `bt_le_adv_start`?

Reply:
Well no, because directed advertising is with `bt_conn_create_slave_le`, and that is only towards one peer.
It will work the same as with `bt_le_adv_start` and `option = BT_LE_ADV_OPT_FILTER_CONN`, then all devices in the whitelist are able to connect.